init-db.yaml

```diff
@@ -0,0 +1,89 @@
+kind: Job
```

Comment:
where is this file being used? should it be added to the template?

Reply:
This is the manual db initialization job.  It only ever needs to be run once against the db after created and isn't yet idempotent.  ATM it's a manual process.